src/test/java/org/apache/commons/compress/archivers/zip/ScatterZipOutputStreamTest.java

```diff
@@ -45,7 +46,8 @@ public void cleanup() {
     @Test
     public void putArchiveEntry() throws Exception {
         scatterFile = File.createTempFile("scattertest", ".notzip");
-        final ScatterZipOutputStream scatterZipOutputStream = ScatterZipOutputStream.fileBased(scatterFile);
+        final ScatterZipOutputStream scatterZipOutputStream = ScatterZipOutputStream.fileBased(scatterFile,
+                Deflater.BEST_COMPRESSION);
         final byte[] B_PAYLOAD = "RBBBBBBS".getBytes();
```

Comment:
Hm, why? How is this part of the NIO change? Hard to te with so many changes...

Reply:
Wanted to increase the coverage and test this branch as well.

Reply:
Please keep the PR focused on one thing, this is a behavior change unrelated to NIO. May you revert this line please?

Reply:
Reverted.